.travis.yml

```diff
@@ -21,9 +27,7 @@ env:
   - NODE_ENV=test DB=mysql IMG=aws
 
 before_install:
-  - npm install -g npm@2
```

Comment:
travis-ci sets the node version with `nvm` so `npm` is installed as the travis user under `/home/travis/.nvm` so overwriting does not require sudo - e.g., https://travis-ci.org/mozilla/fxa-auth-server/jobs/97399693#L256